etl/models/types/model_query.py

```diff
@@ -3,6 +3,6 @@
 from pydantic import Field
 
 # Tiny type for a large language model's question.
```

Comment:
Nit: need to adjust the comment here, and probably use a docstring.

Reply:
I have fixed the comment.